File: 2853/2853.go
func highestSalariesDifference(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := highestSalariesDifference(root.Left)
	r := highestSalariesDifference(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 2853/2853.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var highestSalariesDifference = function(root) {
    if (!root) return 0;
    return 1 + Math.max(highestSalariesDifference(root.left), highestSalariesDifference(root.right));
};